e2e/content-services/directives/delete-directive.e2e.ts

```diff
@@ -0,0 +1,269 @@
+
+            await contentServicesPage.chooseSelectionMode('Multiple');
+            await contentListPage.selectRow(txtFileModel.name);
+            await contentListPage.dataTable.checkRowIsSelected('Display name', txtFileModel.name);
+            await contentListPage.selectRowWithKeyboard(file0BytesModel.name);
+            await contentListPage.dataTable.checkRowIsSelected('Display name', file0BytesModel.name);
+            await contentServicesPage.clickDeleteOnToolbar();
+            await contentListPage.dataTable.checkContentIsNotDisplayed('Display name', txtFileModel.name);
+            await contentListPage.dataTable.checkContentIsNotDisplayed('Display name', file0BytesModel.name);
+        });
+    });
+
+    describe('When selection on multiple pages', () => {
+
+        beforeEach(async () => {
+            await uploadActions.uploadFile( txtFileModel.location, txtFileModel.name, baseFolderUploaded.entry.id);
+            await uploadActions.uploadFile(file0BytesModel.location, file0BytesModel.name, baseFolderUploaded.entry.id);
+            await uploadActions.uploadFile(pdfFileModel.location, pdfFileModel.name, baseFolderUploaded.entry.id);
+            await uploadActions.uploadFile(secondPdfFileModel.location, secondPdfFileModel.name, baseFolderUploaded.entry.id);
+            await uploadActions.uploadFile(pngFileModel.location, pngFileModel.name, baseFolderUploaded.entry.id);
+            await uploadActions.uploadFile(secondPngFileModel.location, secondPngFileModel.name, baseFolderUploaded.entry.id);
+
+            await loginPage.loginToContentServicesUsingUserModel(acsUser);
+            await browser.get(`${browser.baseUrl}/files/${baseFolderUploaded.entry.id}`);
+            await contentServicesPage.waitForTableBody();
+        });
+
+        it('[C260191] - Delete content selected from different pages', async () => {
+            await contentServicesPage.sortByName('ASC');
+            await paginationPage.selectItemsPerPage('5');
+
+            await contentListPage.selectRowWithKeyboard(txtFileModel.name);
+            await contentListPage.dataTable.checkRowIsSelected('Display name', txtFileModel.name);
+            await paginationPage.clickOnNextPage();
+            await contentListPage.selectRowWithKeyboard(file0BytesModel.name);
+            await contentListPage.dataTable.checkRowIsSelected('Display name', file0BytesModel.name);
+            await contentServicesPage.clickDeleteOnToolbar();
+            await contentListPage.dataTable.checkContentIsNotDisplayed('Display name', file0BytesModel.name);
+            await contentListPage.dataTable.checkContentIsDisplayed('Display name', txtFileModel.name);
+        });
+
+    });
+
+    describe('when user does not have `delete` permission', () => {
+
+        let createdSite = null;
+        let fileTxt, filePdf, folderA, folderB;
+
+        beforeEach(async () => {
```

Comment:
creating the site and adding the user as site member needs to be moved in beforeAll, and deleting the site in afterAll